src/components/ClaimReview/form/DynamicForm.tsx

```diff
@@ -182,6 +185,18 @@ const DynamicForm = ({ sentence_hash, personality, claim, sitekey }) => {
         });
 
     const sendEventToMachine = (formData, eventName) => {
+        if (eventName === ReviewTaskEvents.draft) {
+            setIsLoadingDraft(true);
+        } else if (eventName !== ReviewTaskEvents.goback) {
+            setIsLoadingSubmit(true);
+        }
+
+        const setIsLoading =
+            eventName === ReviewTaskEvents.goback
+                ? setIsLoadingGoBack
+                : eventName === ReviewTaskEvents.draft
+                ? setIsLoadingDraft
+                : setIsLoadingSubmit;
```

Comment:
What do you think about extracting this logic to a separate function? 
I'm thinking about something like
```
const chooseLoading = (eventName) => {
        if (eventName === ReviewTaskEvents.draft) {
            return setIsLoadingDraft;
        }
        return eventName === ReviewTaskEvents.goback
            ? setIsLoadingGoBack
            : setIsLoadingSubmit;
    };
```

And then use it like this 

```
const setIsLoading = chooseLoading(eventName);
```

I think this could help with the code smells too.